schema-bridge.graphql

```diff
@@ -310,6 +561,45 @@ type Pool @entity {
   hourlySnapshots: [PoolHourlySnapshot!]! @derivedFrom(field: "pool")
 }
 
+type PoolRoute @entity {
+  # Still thinking about how to make this ID in a way that allows routes to be linked among chains 
+  " {Pool ID}-{lowerChainID}-{greaterChainID} "
+  id: ID!
+
+  " Pool this route belongs to "
+  pool: Pool!
+
+  " Type of the pool on opposite chain "
+  counterType: BridgePoolType
```

Comment:
This seems mistyped, I am not sure what it is

Reply:
Lol, wasn’t sure how to name it. When you send tokens from chain `A` to chain `B`, they are sent from a pool in `A` and received in a pool in `B`. Each of those pools has a type, they go in pairs:
- Lock/Release <> Mint/Burn
- Liquidity <> Liquidity
(not sure if there are more combinations)

`counterType` refers to the type of the pool on the other side of the route.

Reply:
`receiverPoolType`? But that makes sense, so maybe `counterType` is sufficient

Reply:
that should be all the combinations! I hope

Reply:
Added a more descriptive comment on that field then just in case. Kept the same name, since `receiverPoolType` implies that the other pool is a receiver but can also send funds through the route.